vtex/omnishipping
# Fix/without geolocation and postalcode

#### What is the purpose of this pull request?

Fix scenario of a user with a saved address but country doesn't have postal code and we don't have geolocation because data is masked.

#### How should this be manually tested?

1) https://darlene--vtexgame1geo.myvtex.com/checkout/cart/add/?sku=312&qty=1&seller=1&sc=2
2) Identify shipping65.75354206816526@mailinator.com
3) You should be able to select payment options

#### Types of changes
